gamebook/monster.py

```diff
@@ -0,0 +1,28 @@
+from gamebook.creature import Creature
+import random
+
+
+class Monster(Creature):
+    def __init__(self, name, traits):
+        self.name = name
+        self.traits = traits
+        self.is_alive = True
+
+    def attack(self, trait_name):
+        """returns the attack power for the current fight sequence"""
+        for trait in self.traits:
```

Comment:
This for seems to be repeated quite a lot, try to move to the base class? or use as a helper func?